prediction_market_agent/agents/ofvchallenger_agent/deploy.py

```diff
@@ -161,8 +161,8 @@ def challenge_market(
             )
 
         if answer is None:
-            logger.error(
-                f"Failed to get factuality for market {market.url=}, question {market.question_title=}. Skipping."
+            logger.warning(
+                f"OFV didn't factcheck {market.url=}, question {market.question_title=}. Skipping."
```

Comment:
Only case where it returns None is this, that doesn't need to be logged as error:

<img width="545" alt="image" src="https://github.com/user-attachments/assets/2b640455-5218-484c-a840-d9dd3723948b" />